docs/getting-started.md

```diff
@@ -186,6 +186,53 @@ spec:
 EOF
 ```
 
+After a `CloudProviderAccount` CR is added, VPCs are polled from cloud for the
```

Comment:
Also, after a `CloudProviderAccount` CR add, VPCs are automatically polled for the configured region.
Invoke kubectl commands to get the details of imported VPCs.

Reply:
Done